### ISSUE TITLE
Org Tools

### ISSUE THREAD

**Issue by igalic** [fruitfuljs/docs#7]
Following #6, i'd also like to suggest a few meta-tools to manage the org

### Pull Requests & Issues

[peril](https://github.com/danger/peril) allows reacting to the github API events for issues & pull-requests.
It would allow us to automatically react to certain patterns, rather than manually copy/pasting boilerplate responses

### Offering Merge Access

[Moya](https://github.com/Moya/Moya) uses [Aeryn](https://github.com/Moya/Aeryn) to automatically invite everyone who once their first PR has been merged (and they haven't been invited yet)

### What's left?

[Terraform](https://www.terraform.io/docs/providers/index.html) is shaping up to be an API calling API.
We could use it for [github permissions](https://www.hashicorp.com/blog/managing-github-with-terraform/) — but also for our coc@ & security@ mailing lists (although, we'd have make a provider for that;)

**Comment by SomeHats** [comment]
This all looks great.

One question: is there any issue with using both Aeryn and Terraform for GH stuff? I've found terraform can get upset if other stuff modifies what it's trying to control

**Comment by SomeHats** [comment]
Looks like we could use peril to ask people if they want to join when they submit a PR. We could point them at an issue template in our github terraform repo

**Comment by igalic** [comment]
yes, or peril could auto-create a pull-request against our GH-terraform-repo.
Aeryn isn't something we have to use, but i think we should offer commit on pr.
